apps/ts-minbar-test-react-components/assets/patches/README.md

```diff
@@ -0,0 +1,5 @@
+# Patches for TS minbar test
+
+Some partners might be on incompatible versions of typescript, but are willing to patch incompatible
+types on their end. If this is the case, simply create a similar patch for the affected code with
+[patch-package](https://github.com/ds300/patch-package) and drop the resulting patch in this folder.
```

Comment:
can we provide copy pastable command to achieve that ?

Reply:
I can add the command, but it will still involve updating `node_modules` there's no way around that unfortunately